example/kuberhealthy.tf

```diff
@@ -0,0 +1,3 @@
+module "kuberhealthy" {
+  source = "../"
```

Comment:
This might be confusing for an example. Unless it's actually supposed to be this way.

Reply:
Good point Jason. I have amended in later commit.